### PR TITLE
Unskipping tests in CSharp.UnitTests

### PR DESCRIPTION
**Pull request by VSadov**

### CODE REVIEW

**Comment by VSadov** [comment]
@dotnet/roslyn-compiler - please review

**Comment by cston** [comment]
LGTM

File: src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTests.cs [hunk 1]
```diff
@@ -10219,12 +10220,24 @@ static void Main()
...
+  // Code size       11 (0xb)
+  .maxstack  1
+  IL_0000:  newobj     ""Test..ctor()""
+  IL_0005:  call       ""void Test.Foo()""
+  IL_000a:  ret
+}
+");
+
+
+            //NOTE: the resulting code is unverifiable, but we will not insist on that.
```
Comment (AlekseyTs) [inline src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenTests.cs]: It looks like this and two lines above should be deleted.

**Comment by AlekseyTs** [comment]
LGTM, modulo the misplaced comment.

**Comment by VSadov** [comment]
test prtest/mac/dbg/unit32 please
IO error during build

**Pull request merged**